Amazon.QLDB.Driver/client/Session.cs

```diff
@@ -74,7 +74,7 @@ internal class Session
             string ledgerName,
             IAmazonQLDBSession sessionClient,
             ILogger logger,
-            CancellationToken cancellationToken = default)
+            CancellationToken cancellationToken)
```

Comment:
Was this a specific ask to remove the default value? Why?

Reply:
Putting a default value for the CancellationToken is bad practice when it's on levels where it's being passed through as it allows you to have code that ignores it. The default value should only be on entry-level APIs.